content/kubermatic/master/architecture/concepts/global_settings/_index.en.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "Global Settings"
+date = 2018-06-21T14:07:15+02:00
+weight = 5
+
++++
+
+This manual explains how to manage KKP global settings such as custom links, displayed distributions and others.
+It can be done from the UI and also from the command line with `kubectl`.
+
+### Accessing the Admin Panel
+
+Admin panel can be accessed only by logged in users that have administrator privileges. You can check and edit them
+using `kubectl`:
+
+```
+$ kubectl get user -o=custom-columns=INTERNAL_NAME:.metadata.name,NAME:.spec.name,EMAIL:.spec.email,ADMIN:.spec.admin
+$ kubectl edit user ...
+```
+
+Admin privileges can be granted for users also from the admin panel in the UI. To access the admin panel click the
+`Admin Panel` entry in the user menu:
+
+![User menu](/img/kubermatic/master/advanced/settings/menu.png)
+
+### Admin Panel Overview
+
+![Admin panel](/img/kubermatic/master/advanced/settings/view.png)
+
+Admin panel consist of two cards. One allows to manipulate global settings, the second allows to manipulate the list
```

Comment:
the second card now also has dynamic datacenters and presets

Reply:
done